src/storage/Serializer.h

```diff
@@ -0,0 +1,28 @@
+// See the file "COPYING" in the main distribution directory for copyright.
+
+#pragma once
+
+#include "zeek/Span.h"
+#include "zeek/Val.h"
+
+namespace zeek::storage {
+
+namespace detail {
+using byte_buffer = std::vector<std::byte>;
+using byte_buffer_span = Span<const std::byte>;
```

Comment:
Thumbs up for moving into `zeek::util` if you think these are reasonable types for that use case. @bbannier any opinions / experience if that's reasonable aliases in the realm of passing bytes around?